README.md

```diff
@@ -0,0 +1,145 @@
+---
+permalink: /guides/nexus-repository-integration-with-appsody/
+layout: guide-markdown
+title: Nexus Repository Integration
+duration: 40 minutes
+releasedate: 2020-02-21
+description: Learn how to use Nexus Repository in Codewind
+tags: ['Codewind', 'Java', 'Nodejs']
+guide-category:
+---
+# Nexus Repository Integration
```

Comment:
You won't need this h1 as the title: element in the front matter will create it. Otherwise you get 2 headers in the build.

Reply:
commented out